### PR TITLE
[SW-1749] Percentiles Are Not Propagated to Metadata

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/h2o/utils/H2OContextRestAPIUtils.scala [hunk 1]
```diff
@@ -86,7 +86,7 @@ trait H2OContextRestAPIUtils extends H2OContextUtils {
 
   def getFrame(conf: H2OConf, frameId: String): H2OFrame = {
     val endpoint = getClusterEndpoint(conf)
-    val frames = query[FramesV3](endpoint, s"3/Frames/$frameId/light", conf)
+    val frames = query[FramesV3](endpoint, s"3/Frames/$frameId/summary?row_count=0", conf)
```
Comment (jakubhava) [inline core/src/main/scala/org/apache/spark/h2o/utils/H2OContextRestAPIUtils.scala]: 👍

**Comment by jakubhava** [comment]
External tests failing

**Comment by mn-mikke** [comment]
@jakubhava Thanks for letting me know. I've already experienced this problem with `Gson` library. I think the fix would be to try to use  json4s or something else.

**Comment by mn-mikke** [comment]
@jakubhava Can you have a second look?

**Pull request merged**